### PR TITLE
Fix display of amendment from amendment

### LINKED ISSUE #4191: Fix display of amendment from amendment
**Describe the bug**
If an amendment is created on an amendment then inserted text is sometimes not indicated as new text 

**How to Reproduce**
1. Activate the setting "Allow amendment of amendments"
2. Create a motion
3. Create an amendment which creates a new line 
4. In the amendment view: create a new amendment
5. While selecting which lines you want to edit the completely new line is not indicated as such and the paragraph which has a new line there are no numbers behind the line indicator

**The amendment:**
Screenshot_20240930_105852

**Creation mask:**
Screenshot_20240930_105837

**Expected behavior:**
- New lines should be displayed as a inserted line  
- The Indicated line numbers should be the same as they are in the motion

### PR DESCRIPTION
**Pull request by Elblinator**
closes #4191

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/amendment-create-wizard/amendment-create-wizard.component.scss [hunk 1]
```diff
@@ -25,6 +25,10 @@
         }
         .paragraph-text {
             flex: 1;
+
+            .insert {
+                text-decoration: underline !important;
+            }
```
Comment (reiterl) [inline client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/amendment-create-wizard/amendment-create-wizard.component.scss:31]: Mmh, I haven't found the usage of this. Is this debug code?

**Review by reiterl — APPROVED** [review]
LGTM, small remark.

**Review by reiterl — COMMENTED** [review]
Mmh, thought about it some more.
Perhaps use `@let` to remove the code duplication.

File: client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/amendment-create-wizard/amendment-create-wizard.component.scss (hunk 1 above)
Comment (Elblinator) [reply client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-form/components/amendment-create-wizard/amendment-create-wizard.component.scss:31]: This fixes the missing underline if the previous amendment created a completely new paragraph
So no debug code :D

**Review by reiterl — APPROVED** [review]
Okay, that is better, I think.

**Review by MSoeb — APPROVED** [review]
Looks fine.

**Pull request merged**